integreat_cms/cms/templates/linkcheck/links_by_filter.html

```diff
@@ -173,4 +173,63 @@ <h1 class="heading">
     </form>
     {% url 'linkcheck' as url %}
     {% include "pagination.html" with url=url chunk=page_obj %}
+    {% if request.user.expert_mode %}
+        <div class="flex-wrap relative w-auto mt-12">
+            <h3 class="font-bold text-xl">{% translate "Replace region link" %}</h3>
+            <div class="pt-4 pb-4">
+                <span class="font-bold pb-2">{% translate "Here you can replace internal links, e-mail addresses and phone numbers partially or completely." %}</span>
+                <div class="pl-4">
+                    <span>{% translate "Example 1: " %}</span>
+                    <span class="bg-red-300">/augsburg/</span> -> <span class="bg-green-300">/nurnberg/</span>
+                    <div class="pl-10">
+                        https://integreat.app<span class="bg-red-300">/augsburg/</span>de/willkommen-in-augsburg -> https://integreat.app<span class="bg-green-300">/nurnberg/</span>de/willkommen-in-augsburg
+                    </div>
+                </div>
+                <div class="pl-4">
+                    <span>{% translate "Example 2: " %}</span>
+                    <span class="bg-red-300">augsburg</span> -> <span class="bg-green-300">nurnberg</span>
+                    <div class="pl-10">
+                        https://integreat.app/<span class="bg-red-300">augsburg</span>/de/willkommen-in-<span class="bg-red-300">augsburg</span> -> https://integreat.app/<span class="bg-green-300">nurnberg</span>/de/willkommen-in-<span class="bg-green-300">nurnberg</span>
+                    </div>
+                </div>
+                <div class="pl-4">
+                    <span>{% translate "Example 3: " %}</span>
+                    <span class="bg-red-300">/augsburg/de/willkommen</span> -> <span class="bg-green-300">/augsburg/de/hallo</span>
+                    <div class="pl-10">
+                        https://integreat.app<span class="bg-red-300">/augsburg/de/willkommen</span>-in-augsburg -> https://integreat.app<span class="bg-green-300">/augsburg/de/hallo</span>-in-nurnberg
+                    </div>
+                </div>
+                <span class="font-bold">{% translate "External links will not be changed." %}</span>
+            </div>
+            <div class="flex mt-2 mb-2 gap-2">
+                <input id="replace_region_from"
```

Comment:
You could also define these input fields via a real Django form and render them with `{% render_field %}` and perform the replacement step inside the form's `save()` method - I guess this would be a little more Django-standard...